package.json

```diff
@@ -27,7 +37,19 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
```

Comment:
We were planning on using stuff like localForage & axios for the client, which needs a bundler. Babelify is to transpile the modules to es5 to allow a greater browser support with less effort from our side.

If you'd like to see some over the top sh*t, please checkout [@Mimaaa's Webpack setup](https://github.com/rijkvanzanten/obachine/blob/master/webpack.parts.js)